text_extensions_for_pandas/io/watson.py

```diff
@@ -0,0 +1,136 @@
+#
+#  Copyright (c) 2020 IBM Corp.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+################################################################################
+# watson.py
+#
+# I/O functions related to Watson Natural Language Processing on the IBM Cloud.
+
+import pandas as pd
+import pyarrow as pa
+
+from text_extensions_for_pandas.array import CharSpanArray, TokenSpanArray
+
+
+def watson_nlp_parse_response(response):
+
+    def flatten_struct(struct_array, parent_name=None):
```

Comment:
These inline functions should probably be moved to the top-level scope of this file and renamed to start with an underscore so they don't get picked up by `__all__`. That way the source code of the `watson_nlp_parse_response()` function will come closer to fitting on a single screen.